pkg/client/cmd/login_cluster.go

```diff
@@ -30,6 +37,7 @@ func login(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
+	fmt.Printf("expiresin: #v float: %f\n", expiresIn, float64(expiresIn))
```

Comment:
Could you remove this print? (Is it a debug print?)

Reply:
😮 
removed